src/main/java/com/github/javaparser/matchersj/Matchers.java

```diff
@@ -0,0 +1,49 @@
+package com.github.javaparser.matchersj;
+
+import com.github.javaparser.ast.Node;
+import com.github.javaparser.matchersj.classes.*;
+import com.github.javaparser.symbolsolver.javaparser.Navigator;
+
+import java.util.List;
+import java.util.stream.Collectors;
+
+/**
+ * Created by cdo on 6/15/17.
+ *
+ * Most static methods in this class are a bare call to its classes
+ * This was written in order to get better code elegancy when writting matchers
```

Comment:
writting -> typo

Yes, I think these methods are a really good idea: we need those. We should just rename all of them to start with a lowercase letter